examples/ex1.cpp

```diff
@@ -66,6 +66,8 @@ int main(int argc, char *argv[])
    int order = 1;
    bool static_cond = false;
    bool pa = false;
+   bool ea = false;
+   bool fa = false;
```

Comment:
Might want to use an enum here instead of bools since these options are mutually exclusive.

Reply:
I agree with you, but we had this discussion before on #1340 and the powers that be decided on bools.

Reply:
They're gone.